### PR TITLE
Added script for automatically download all needed models at install

### PR DESCRIPTION
**Pull request by ATAMAH**
# PR type

new feature

# Description

Automatically download all needed models at installation from https://huggingface.co/lj1995/VoiceConversionWebUI/tree/main

Just run:

`python download_models.py`

### CODE REVIEW

**Comment by blaise-tk** [comment]
This is interesting, but it could be even better if you make it so that when you start RVC, it checks if those files exist and if they don't exist, it downloads them. We did that in Applio-RVC-Fork and it saves a lot of trouble and makes everything easier!

**Comment by ATAMAH** [comment]
> This is interesting, but it could be even better if you make it so that when you start RVC, it checks if those files exist and if they don't exist, it downloads them. We did that in Applio-RVC-Fork and it saves a lot of trouble and makes everything easier!

I can update it next commit. It is good for now cause the only way to install these models is manually downloading.

**Review by fumiama — CHANGES_REQUESTED** [review]
Good job, but you can move this file into `tools` folder since there are already two auto-downloading scripts.

**Comment by ATAMAH** [comment]
> Good job, but you can move this file into `tools` folder since there are already two auto-downloading scripts.

Done.

File: docs/en/README.en.md [hunk 1]
```diff
@@ -97,23 +97,9 @@ sh ./run.sh
 ## Preparation of other Pre-models
 RVC requires other pre-models to infer and train.
 
-You need to download them from our [Huggingface space](https://huggingface.co/lj1995/VoiceConversionWebUI/tree/main/).
```
Comment (fumiama) [inline docs/en/README.en.md:100]: Well, you can keep this instruction if someone would like to download the files themselves. Just add the python instruction at the top.

**Review by fumiama — CHANGES_REQUESTED** [review]

File: docs/en/README.en.md (hunk 1 above)
Comment (ATAMAH) [reply docs/en/README.en.md:100]: Check.

**Pull request merged**